tests/environment.py

```diff
@@ -0,0 +1,175 @@
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2020 Arnout Vandecappelle (Essensium/Mind)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+import json
+import os
+import re
+import subprocess
+
+from capi import UCCSocket
+from opts import debug
+import sniffer
+
+
+class AlEntity:
```

Comment:
nitpick: maybe make the class name `ALEntity`, to avoid confusion (I originally read it as "A1 entity")

Reply:
Not really PEP8 naming but probably a good idea indeed.